docs/source/tutorial.rst

```diff
@@ -294,3 +294,77 @@ The stack of DataPipes can then be constructed using their functional forms (rec
 
 In the above example, ``datapipes1`` and ``datapipes2`` represent the exact same stack of ``IterDataPipe``\s. We
 recommend using the functional form of DataPipes.
+
+Working with Cloud Storage Providers
+---------------------------------------------
+
+Accessing AWS S3 with ``fsspec`` DataPipes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This requires the installation of the libraries ``fsspec``
+(`documentation <https://filesystem-spec.readthedocs.io/en/latest/>`_) and ``s3fs``
+(`s3fs GitHub repo <https://github.com/fsspec/s3fs>`_).
+
+You can list out the files within a S3 bucket directory by passing a path that starts
+with ``"s3://BUCKET_NAME"`` to ``FSSpecFileLister``.
+
+.. code:: python
+
+    from torchdata.datapipes.iter import IterableWrapper
+
+    dp = IterableWrapper(["s3://BUCKET_NAME"]).list_files_by_fsspec()
+
+You can also open files using ``FSSpecFileOpener`` and stream them (if supported by the file format).
```

Comment:
Where are we using FSSpectFileOpener and Saver?

Reply:
Edited to mention that is equivalent to `.open_files_by_fsspec(...)`